Code_Of_Conduct.md

```diff
@@ -0,0 +1,74 @@
+# Code of Conduct
+
+----
+## Our Pledge
+
+We as project administrators and mentors pledge to make participation and contribution in our community a harassment-free experience for all, regardless of race, gender, sexual orientation, age, education, experience level, nationality, religion, disability or body size. 
+
+We pledge to always act and interact in ways that foster an inclusive, diverse, open, welcoming and healthy community.
+
+## Our Standards
+
+Examples of unacceptable behavior include: 
+
+* The use of sexually charged language or imagery, including sexual advances of any kind. 
+* Derogatory comments, personal or political attacks. 
+* Publishing others' private information such as physical address or email address without their express permission.
+* Public or private harassment of any kind.
+* Any other conducts that could be considered inappropriate in a professional setting. 
+
+Harassment includes but is not limited to offensive verbal or written comments related to gender, age, sexual orientation, disability, physical appearance, body size, race, religion,
+social class, economic status, veteran status, sexual images, deliberate intimidation,stalking, following, harassing \ photography or recording, sustained disruption of talks or
+other events, inappropriate physical contact, and unwelcome sexual attention. If what you’re doing is making someone feel uncomfortable, that counts as harassment and is enough reason to stop doing it.
+
+
+
+Examples of behavior that contributes to a positive environment for our community:
+
+* Showing kindness and empathy toward other people. 
+* Using respectful, welcoming and inclusive language. 
+* Providing and receiving feedback gracefully. 
+* Accepting responsibility and apologizing to those affected by our mistakes.
+* Focusing on what is best not just for us as individuals but for the overall community.
+
+## Enforcement Responsibilities
+
+Project maintainers are responsible for clarifying and enforcing standards of acceptable behavior and would take appropriate and fair corrective action in response to any inappropriate behavior.
+
+Project maintainers have the right to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned with this Code of Conduct, and will communicate reasons for moderation decisions when appropriate. 
+
+## Scope
+
+This Code of Conduct applies to project administrators, mentors and all contributors within all community spaces.
+
+## Enforcement
+
+Instances of harassment, abuse or any unacceptable behavior may be reported to project administrators responsible for enforcement by emailing (  ). All complaints will be reviewed and investigated promptly and fairly. 
```

Comment:
@bjost2s any suggestions what mail-address to use here?

Reply:
@bjost2s friendly ping

Reply:
I think we should use our general support email adress which is support-o-r@iais.fraunhofer.de
@ensonic , @Yuvraj-kadale , sorry for the delay